docs/UserGuide.adoc

```diff
@@ -54,10 +54,101 @@ e.g. typing *`help`* and pressing kbd:[Enter] will open the help window.
 === Recipe inventory commands
 
 === Database commands
```

Comment:
Maybe can rename to "Cookbook commands" as users shouldn't need to know what the database is?